server/characters/characters_common.py

```diff
@@ -29,4 +32,27 @@ def db_connection():
     if config.has_option('Characters','characters_psql_pass'):
         db_pass = config.get('Characters','characters_psql_pass')
     connection = psycopg2.connect("dbname=%s user=%s password=%s" % (db, username,db_pass))
-    return connection
\ No newline at end of file
+    return connection
+
+def fetchall_from_db_query(query):
+    """Run a query on the character database and return all of the results."""
+    connection = db_connection()
+    myCursor = connection.cursor()
+    myCursor.execute(query)
```

Comment:
My statically-typed brain is shivering at the lack of input type checking. But that is how python works.